lib/pf/mfa/TOTP.pm

```diff
@@ -66,6 +67,8 @@ sub verify_otp {
     if (defined $person->{otp} && $person->{otp} ne '') {
         my $local_otp = $self->generateCurrentNumber($person->{otp});
         if ($otp == $local_otp) {
+            #MFA verification success
+            cache->set($username."mfapostauth", $TRUE, normalize_time($self->post_mfa_validation_cache_duration));
```

Comment:
I'm going to be mr. picky but this is copy/pasted between the 2 MFA modules, please centralize the logic of success/failure MFA calls in the cache to the base MFA module